jetty-12 ee8 JakartaWebSocketShutdownContainer NotFoundException

**jetty-12**

Running the `ee8` demos you get `INFO` but nonetheless error messages from the `AnnotationConfiguration` class for every `ee8` webapp that is deployed, here's one example:

> 2022-10-10 17:49:01.408:INFO :oejea.AnnotationConfiguration:main: Error:  for o.e.j.e.w.WebAppContext@a137d7a{ee8-demo-proxy,/ee8-demo-proxy,file:///home/janb/src/jetty-eclipse/jetty-12.0.x/jetty-home/target/base/work/jetty-0_0_0_0-8443-ee8-demo-proxy_war-_ee8-demo-proxy-any-/webapp/,false}{/home/janb/src/jetty-eclipse/jetty-12.0.x/jetty-home/target/base/webapps/ee8-demo-proxy.war}

With some extra debugging in there, we see that all of the errors are caused by this:

> Caused by: java.util.ServiceConfigurationError: javax.servlet.ServletContainerInitializer: Provider org.eclipse.jetty.ee8.websocket.javax.client.JakartaWebSocketShutdownContainer not found
> 	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)
> 	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1219)
> 	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228)
> 	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
> 	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
> 	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
> 	at org.eclipse.jetty.util.ServiceLoaderSpliterator.tryAdvance(ServiceLoaderSpliterator.java:37)

Worth noting that this error does _not_ occur for `ee9`, from which `ee8` is a direct translation (nor FWIW `ee10`).

--- a/jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod
+++ b/jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod
@@ -16,12 +16,12 @@ lib/ee8-websocket/jetty-websocket-core-common-${jetty.version}.jar
 lib/ee8-websocket/jetty-websocket-core-client-${jetty.version}.jar
 lib/ee8-websocket/jetty-websocket-core-server-${jetty.version}.jar
 lib/ee8-websocket/jetty-ee8-websocket-servlet-${jetty.version}.jar
-lib/ee8-websocket/jetty-jakarta-websocket-api-@jakarta.websocket.api.version@.jar
+lib/ee8-websocket/jetty-javax-websocket-api-@jakarta.websocket.api.version@.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-client-${jetty.version}.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-common-${jetty.version}.jar
-lib/ee8-websocket/jetty-ee8-websocket-jakarta-server-${jetty.version}.jar
+lib/ee8-websocket/jetty-ee8-websocket-javax-server-${jetty.version}.jar
 
 [jpms]
 # The implementation needs to access method handles in
 # classes that are in the web application classloader.
-add-reads: org.eclipse.jetty.websocket.jakarta.common=ALL-UNNAMED
+add-reads: org.eclipse.jetty.websocket.javax.common=ALL-UNNAMED

